### ISSUE TITLE
_hasPhoneAttached() fails and do not return true data. 

### ISSUE THREAD

**Issue by JabLuszko** [geel9/SteamAuth#14]
string response for _hasPhoneAttached() is
```
 {"success":false,"error_text":"Missing or invalid form session key","fatal":false}
```
so it ends with HasPhone as false, always.

**Comment by Ne3tCode** [comment]
[Recent changes](https://github.com/SteamDatabase/SteamTracking/commit/911a56b6d6bf3da5e24a2a4ef74a5d59002e72ed/Scripts/Community/steamguard.js)
#13 #15

**Comment by geel9** [comment]
Fixed in c4745e365b91205d3b86f3b69e0981da31ee9c44
